R/merge.R

```diff
@@ -223,15 +223,15 @@
     x
 }
 
-.merge_cols_TSE <- function(x, f, archetype = 1L, mergeTree = FALSE, ...){
+.merge_cols_TSE <- function(x, f, archetype = 1L, agglomerate.tree = FALSE, ...){
     # input check
-    if(!.is_a_bool(mergeTree)){
-        stop("'mergeTree' must be TRUE or FALSE.", call. = FALSE)
+    if(!.is_a_bool(agglomerate.tree)){
+        stop("'agglomerate.tree' must be TRUE or FALSE.", call. = FALSE)
     }
     #
```

Comment:
Same here. Check if deprecation is taken care of. (unit tests should reveal that)

Reply:
ACK.